opm/grid/common/ZoltanGraphFunctions.hpp

```diff
@@ -167,7 +167,7 @@ class CombinedGridWellGraph
     double edgeWeight(int face_index) const
     {
         if (edgeWeightsMethod_ == uniformEdgeWgt)
-            return 1.0;
+            return transmissibilities_[face_index] == 0.0 ? 0.0 : 1.0;
```

Comment:
Here we seem to rely on transmissibilties being there, too. That seems a bit odd for uniform weights. Isn't transmissibilities even a nullptr if loadbalance is called without it.